Tools/build.bat

```diff
@@ -189,6 +190,34 @@ set __VERBOSITY__=n
 msbuild.exe %BUILD_FILE% /t:%__TARGET__% /p:Configuration="%~1" /P:Platform=%2 -fileLoggerParameters:Verbosity=%__VERBOSITY__%;LogFile=%BUILD_LOG_FILE%
 goto :eof
 
+:run_semmle
+if "%SEMMLE_HOME%"=="" (
+  echo "Semmle path is not configured, skipping"
+  goto :eof
+)
+
+echo "Removing previously created rules database"
+rmdir /s/q semmle_db
+
+echo "Creating rules database for %BUILD_FILE%"
+echo msbuild.exe %BUILD_FILE% /t:rebuild /p:Configuration="%~1" /P:Platform=%2 > semmle_bld.bat
+call %SEMMLE_HOME%\codeql\codeql.exe database create -l=cpp -c "semmle_bld.bat" semmle_db -j 0
```

Comment:
I think it better uses the current project as a `semmle_db` name. This allows saving all codeql logs in case of buildAll was called.

This is my draft for codeql https://github.com/kostyanf14/kvm-guest-drivers-windows/commit/a20aec94a98eeee52ef2d47fc5ff468d3fc31119

Reply:
But it seems to me that for building individual drivers and for cleanup it is more convenient to create a codeql base inside the driver directory as SDV does.

Reply:
It makes sense. I agree.